.github/workflows/llvm-project-tests.yml

```diff
@@ -34,9 +34,7 @@ on:
         type: string
         # Use windows-2019 due to:
         # https://developercommunity.visualstudio.com/t/Prev-Issue---with-__assume-isnan-/1597317
-        # We're using a specific version of macOS due to:
-        # https://github.com/actions/virtual-environments/issues/5900
-        default: '["ubuntu-latest", "windows-2019", "macOS-11"]'
+        default: '["ubuntu-latest", "windows-2019", "macOS-latest"]'
```

Comment:
We should use `macOS-12` rather than `macOS-latest` so we have control over when we update to the next version.

Reply:
done.